dataflow/dags/tags_classifier_pipeline.py

```diff
@@ -0,0 +1,64 @@
+from dataflow.operators.tags_classifier_model.tags_classifier import make_prediction
+import sqlalchemy as sa
+from airflow.operators.python_operator import PythonOperator
+from dataflow.dags import _PipelineDAG
+from dataflow.utils import TableConfig
+from dataflow.dags.dataset_pipelines import InteractionsDatasetPipeline
+
+
+class TagsClassifierPipeline(_PipelineDAG):
+    table_config = TableConfig(
+        table_name='interactions_dataset_with_tags',
```

Comment:
```suggestion
        schema='dit',
        table_name='interactions_dataset_with_tags',
```

There's also a `schema` parameter here that might be good to use - probably it should just be `dit`.